ui/widgets/datepicker.js

```diff
@@ -61,12 +62,13 @@ var widget = $.widget( "ui.datepicker", {
 
 	_create: function() {
 		this.suppressExpandOnFocus = false;
+		this._parse = new Globalize( this.options.locale ).dateParser( this.options.dateFormat );
 
 		if ( $.type( this.options.max ) === "string" ) {
-			this.options.max = Globalize.parseDate( this.options.max, { raw: "yyyy-MM-dd" } );
+			this.options.max = this._parse( this.options.max );
```

Comment:
With the parsing happening in `_getCreateOptions`, I'm even more confused, since that uses a different format.

Since I doubt these changes are happening by accident, let's update the spec.